sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyServiceVersion.java

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.keys;
+
+import com.azure.core.http.ServiceVersion;
+
+/**
+ * The versions of Azure Key Vault supported by this client library.
+ */
+public enum KeyServiceVersion implements ServiceVersion {
+    V7_0("7.0");
```

Comment:
It might pay to confer with @schaabs about whether the service versions of Key Vault will always move in lock-step (for keys, secrets, and certs), or if there is a possibility that they may be versioned independently.

Reply:
They have in the past, but not necessarily moving forward. They are adding another resource type which will share the Keys protocol but doesn't implement the Secrets or Certificates protocols. Given that it's somewhat likely that they would start to version these separately.